jetson/percep/camera.hpp

```diff
@@ -1,6 +1,8 @@
 #pragma once
 #include <opencv2/opencv.hpp>
 #include "perception.hpp"
+#include "rapidjson/document.h"
```

Comment:
Could you include these in perception.hpp? Should keep you from having to repeatedly include them in other files

Reply:
Did you try this and it not work? It looks like you deleted it and then added it back in. Could you add it to Perception.hpp?

Reply:
I have it in both files, but for some reason when I remove it from camera.cpp it doesn't compile

Reply:
That's strange. We can just leave it for now